dcc-storage-server/src/main/java/org/icgc/dcc/storage/server/controller/ObjectUploadController.java

```diff
@@ -68,7 +75,7 @@
       HttpServletRequest request) {
 
     log.info("Request received from client {}", userAgent);
-    String ipAddress = request.getHeader(HttpHeaders.X_FORWARDED_FOR);
+    String ipAddress = request.getHeader(com.google.common.net.HttpHeaders.X_FORWARDED_FOR);
```

Comment:
Please lets not repeat these (seen at least 3 times). Perhaps make a utility or put in a base class. Thanks.